src/IonBinaryReader.ts

```diff
@@ -18,22 +18,20 @@
 // Handles system symbols, and conversion from the parsed
 // input byte array  to the desired Javascript value (scalar or
 // object, such as IonValue).
-import { Catalog } from "./IonCatalog";
-import { Decimal } from "./IonDecimal";
-import { defaultLocalSymbolTable } from "./IonLocalSymbolTable";
-import { getSystemSymbolTable } from "./IonSystemSymbolTable";
-import { Import } from "./IonImport";
-import { ion_symbol_table_sid } from "./IonSymbols";
-import { IonType } from "./IonType";
-import { IonTypes } from "./IonTypes";
-import { IVM } from "./IonConstants";
-import { LocalSymbolTable } from "./IonLocalSymbolTable";
-import { makeSymbolTable } from "./IonSymbols";
-import { ParserBinaryRaw } from "./IonParserBinaryRaw";
-import { Reader } from "./IonReader";
-import { SharedSymbolTable } from "./IonSharedSymbolTable";
-import { BinarySpan } from "./IonSpan";
-import { Timestamp } from "./IonTimestamp";
+import {Catalog} from "./IonCatalog";
+import {Decimal} from "./IonDecimal";
+import {defaultLocalSymbolTable, LocalSymbolTable} from "./IonLocalSymbolTable";
+import {ion_symbol_table_sid, makeSymbolTable} from "./IonSymbols";
+import {IonType} from "./IonType";
+import {IonTypes} from "./IonTypes";
+import {IVM} from "./IonConstants";
+import {ParserBinaryRaw} from "./IonParserBinaryRaw";
+import {Reader} from "./IonReader";
+import {BinarySpan} from "./IonSpan";
+import {Timestamp} from "./IonTimestamp";
+import JSBI from "jsbi";
+import IntSize from "./IntSize";
+import {JsbiSupport} from "./JsbiSupport";
```

Comment:
This seems like formatting changes. Did we pin down a formatter? It would be nice to avoid these sort of changes in the diffs.

Reply:
> This seems like formatting changes. Did we pin down a formatter?

We were holding off on selecting a formatter until we had migrated the tests to Typescript, which this PR completes. This change, however, came from my opinionated IDE and was not intentional. 

>  It would be nice to avoid these sort of changes in the diffs.

Agreed, sorry about that.